src/LibraryManager/Providers/Unpkg/UnpkgCatalog.cs

```diff
@@ -178,45 +178,52 @@ private void GetFiles(JObject fileObject, List<string> files)
 
             (string name, string version) = LibraryNamingScheme.Instance.GetLibraryNameAndVersion(libraryNameStart);
 
+            int at = name.IndexOf('@');
+            name = at > -1 ? name.Substring(0, at) : name;
```

Comment:
This should also handle having an @ at the beginning of e.g. a scoped package name.

Reply:
Start searching at the second character.

---
In reply to: [199293261](https://github.com/aspnet/LibraryManager/pull/258#discussion_r199293261) [](ancestors = 199293261)